### PR TITLE
New documentation 

### PR DESCRIPTION
**Pull request by gisantin**
**CHANGELOG** :memo:

* Create and configure documentation using Storybook. 
* Remove old docs 
* Create initial components for new ID

**Prints**

Screen Shot 2019-11-27 at 16 47 18

Screen Shot 2019-11-27 at 16 48 14

### CODE REVIEW

File: src/components/button/index.scss [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+    &:hover {
+      background-color: $color-brand-first-medium;
+    }
+  }
+
+  &__clear--default {
+    color: $color-interface-content-extra-dark;
+    font-size: 12px;
+    padding: 11px 24px;
```
Comment (pyymenta) [inline src/components/button/index.scss]: Could we have a token here:

$size-spacing-s = 24px

File: .storybook/addons.js [hunk 2]
```diff
@@ -0,0 +1 @@
+import '@storybook/addon-viewport/register';
```
Comment (pyymenta) [inline .storybook/addons.js]: Storybook :book: :blue_book: 

:clap: :clap: :clap: :clap: :clap: :clap: :clap: :clap:

Comment (yvessacurae) [inline .storybook/addons.js]: ❤️ ❤️ ❤️ ❤️ ❤️

**Comment by gisantin** [comment]
> Aquele logo da GetNinjas faltando no print vai ser corrigido?

Vai sim!! =]

**Pull request merged**